napari/utils/_magicgui.py

```diff
@@ -123,49 +241,47 @@ def show_layer_result(gui, result: Any, return_type: Type[Layer]) -> None:
     if result is None:
         return
 
+    # This is the pre 0.4.3 API, warn user and pass to the correct function.
```

Comment:
Do we want to say something like "this behavior will no longer be supported after napari 0.4.5" in these messages

Reply:
in the comment? or in the warning?  (I felt like that warning was already approaching a book chapter 😂 )

Reply:
i meant in warning, we've tended to be explicit lately and i think it helps set expectations

Reply:
could make it `0.4.6` if you want longer